docs/README.md

```diff
@@ -0,0 +1,22 @@
+# Contributing to documentation
+
+## General references
+
+Our docs run on a distribution of [`mkdocs`](https://www.mkdocs.org/) called [`mkdocs-material`](https://squidfunk.github.io/mkdocs-material/).
+
+For basics of doc writing, see the [Writing your docs](https://www.mkdocs.org/user-guide/writing-your-docs/) section of `mkdocs` user guide.
+
+To preview as you write, use `mkdocs serve` (after having installed pip's development requirements and `cd`'ed to this directory). That will start a hot-reloading development server.
```

Comment:
I predict that we may need to flesh these instructions out at some point, providing some clearer steps, akin to these [steps that I wrote for CiviCRM](https://docs.civicrm.org/dev/en/latest/documentation/#mkdocs). But we can see if anyone gets confused and add more clarity later if needed.